pkgs/games/freeorion/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchgit, cmake, boost, SDL2, python2, freetype, openal, libogg, libvorbis, zlib, libpng, libtiff, libjpeg, mesa, glew, git, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "2016-04-16";
+  name = "freeorion-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/freeorion/freeorion";
```

Comment:
strace was how we got to the font theory originally. It's the DejaVuSans and DejaVuSans-Bold font inside the game's directory (and that come with the game) that keep getting opened.

The weird thing is that it seems to access the font in both of the derivations I have in /nix/store/. I'm quite sure that's not supposed to happen.